keyboards/hexboard/1_1/rules.mk

```diff
@@ -0,0 +1,10 @@
+SRC += matrix.c
+CUSTOM_MATRIX = lite
+
+RGB_MATRIX_CUSTOM_KB = yes
+
+## for both SH1106/SH1107
+OLED_DRIVER = ssd1306
+OLED_TRANSPORT = i2c
```

Comment:
These can be removed, since they are the default behavior for the oled driver feature.